src/model/codacyInput.ts

```diff
@@ -30,12 +30,16 @@ export class Tool {
   }
 }
 
+export type Options = { [k: string]: any }
```

Comment:
Should limit it to `string`,  `number` and `boolean`. Use `any` as little as possible.

Reply:
according to the specification, Codacy can send any value here, not just `string`, `number` or `boolean`.

Reply:
Unexpected.